en/dev_log/ulog_file_format.md

```diff
@@ -41,6 +45,9 @@ The file consists of three sections:
 ----------------------
 ```
 
+Description on each section is explained below.
```

Comment:
Note, this says in English "below I will explain the description" not "below I will explain the section".

Reply:
Ohhh yes I am Explaining the Description :rofl:, my english is :camel:

Reply:
Thank you for the catch!